src/modules/displayList.js

```diff
@@ -4,12 +4,13 @@ export default (todoObjects) => todoObjects.map((todo) => {
   li.setAttribute('data-index', todo.index);
   li.setAttribute('data-completed', todo.completed);
```

Comment:
- [ ] I couldn't check requested changes as your function for adding a new task isn't working right now. Kindly fix it. It gives this console error:
![image](https://user-images.githubusercontent.com/91486791/220937814-38f1aada-c1bb-42ad-a132-b15c1653f383.png)

Reply:
Hi @tugsem :wave: 
Thank you for your feedback. 
Fixed, please [have a look](https://github.com/ZewdieMc/To-Do-List/pull/2/commits/0d46c44b4806367035bfa5b34974879a988d01fe).